### PR TITLE
added isTaxDeductable to Accounts

### PR DESCRIPTION
**Pull request by delianides**

### CODE REVIEW

**Review by jbaxleyiii — APPROVED** [review]

**Comment by jbaxleyiii** [comment]
@delianides I think this is a good change, but looking at the issue, this actually wont solve it.

The issue is with giving history both on web and in print. The methods that need adapting are:
https://github.com/NewSpring/Heighliner/blob/master/src/rock/models/finances/resolver.js#L142 and
https://github.com/NewSpring/Heighliner/blob/master/src/rock/models/finances/resolver.js#L26-L37

**Review by jbaxleyiii — CHANGES_REQUESTED** [review]
We need to fix the pulling of transactions, not just the pulling of accounts

File: src/rock/models/finances/models/Transaction.js [hunk 1]
```diff
@@ -168,6 +178,7 @@ export default class Transaction extends Rock {
         order: [["TransactionDateTime", "DESC"]],
         where: TransactionDateTime ? [{ TransactionDateTime }] : null,
         include: [
+          { model: TransactionDetail.model, where: { AccountId: { $in: deductibleAccounts } } },
```
Comment (JakeDawkins) [inline src/rock/models/finances/models/Transaction.js:181]: @jbaxleyiii this is the only one I'm kinda unsure about, since it has multiple includes

Comment (jbaxleyiii) [reply src/rock/models/finances/models/Transaction.js:181]: @JakeDawkins that should work great as you have it written

**Review by jbaxleyiii — APPROVED** [review]

**Pull request merged**